src/hasReachedStyle.js

```diff
@@ -1,4 +1,6 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+export default function hasReachedStyle(currentStyle: Object,
```

Comment:
Humm on second thought it could be a bit complicated. The thing is, `currentStyle` looks like this: `{height: {val: 10, config: [1, 2]}, width: justANumber, randomKey: someRandomValue}`. The `randomKey` part screws things up. It's needed because for TransitionMotion you might need to carry around a pack of info around so that you can still access them when the item's gone everywhere else. Do you know what I mean?

If this is type-checkable it'd be great. Say it's type A. TransitionMotion's `styles` would be typed as a map of `{stringKey: A}`. Avoids some confusion if you use types everywhere.

So right now it's not easily typeable. 2 solutions I can think of:
1. Impose that recognized keys, e.g. `height`, `transform`, `opacity`, etc. Have that Number | ConfigObject type. This isn't cross-platform though.
2. Impose that anything you carry around be in a key called `data`, of type `Any`. Everything else should be Number | ConfigObject.

Any other solution? If not, either way we'd need to leave this as `Object` for now.